concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,221 @@
+Security Advisories"
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported by email to security@enarx.io. 
+This list is delivered to a small security team. Your email will be 
+acknowledged within 48 hours, and you’ll receive a more detailed response to 
+your email within 96 hours indicating the next steps in handling your report.
+ If you wish, you may encrypt your report using our public key. This key is 
+also on keys.openpgp.org and reproduced below.
+
+Please ensure that you use a descriptive subject line to avoid having your report be 
+missed (as with any publicly listed addresses, it is expected this one will 
+receive a lot of spam). After the initial reply to your report, the security team
+ will endeavor to keep you informed of the progress being made towards a fix 
+and full announcement. As recommended by 
+[RFPolicy](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt),
+ these updates will be sent at least every five working days.
+
+If you have not received a reply to your email within 72 hours, or have not 
+heard from the security team for the past five days, there are a few steps you
+ can take (in suggested order):
+- Contact the current security coordinator (Nathaniel McCallum (public key)) directly.
+- Contact the back-up contact (Mike Bursell (public key)) directly.
+- Come chat to us on the [chat](https://gitter.im/enarx/community).
+
+Please note that the chat is a public area. When escalating in these venues,
+please do not discuss your issue. 
+Simply say that you’re trying to get a hold of someone from the security team.
+
+Should you need to transfer files to us in a private manner, please use 
+[Magic Wormhole](https://magic-wormhole.readthedocs.io/en/latest/) or make 
+them available over https.
+
+
+#### Disclosure policy
+The Enarx project has a 5 step disclosure process.
+1. The security report is received and is assigned a primary handler. 
+  This person will coordinate the fix and release process.
+2. The problem is confirmed and a list of all affected versions is determined.
+3. Code is audited to find any potential similar problems.
+4. Fixes are prepared for all releases which are still under maintenance. 
+  These fixes are not committed to the public repository but rather held 
+  locally pending the announcement.
+5. On the embargo date, an announcement is sent to Enarx public channels 
+  (chat, website). The changes are pushed to the public repository and new 
+  builds are deployed.
```

Comment:
This seems to imply there will always be an embargo, but the embargo section states that embargoes should be used sparingly. Perhaps adding details on what a non-embargo process looks like and contrasting it would be useful here, ex. `5. If under embargo, on the embargo date....`

Or maybe just changing this section name to `Disclosure policy in case of embargo`?

Reply:
Thanks, well spotted. There should really be one procedure, with minor variations in case of embargo. Reworking it to incorporate that.